helper/schema/schema.go

```diff
@@ -201,6 +201,23 @@ func (s *Schema) GoString() string {
 	return fmt.Sprintf("*%#v", *s)
 }
 
+// Returns a default value for this schema by either reading Default or
+// evaluating DefaultFunc. If neither of these are defined, returns nil.
+func (s *Schema) GetDefaultValue() (interface{}, error) {
```

Comment:
More idiomatic Go to drop the `Get`, the getter/setter model of Go is `SetX(value)` and `X() value`